### DISCUSSION TITLE
This looks interesting, what are you building?

### DISCUSSION THREAD

**Discussion by ConorSAI** [cubiq/Mellon#3] [General]
Hi there,  I stumbled across this and my curiosity is piqued around this project.

I am mainly wondering what you are building,  specifically what this is compared to comfyui?  How do you plan it to be different?

I have a lot of end-user ideas of what you might be after,  but I see this is very new from the looks of it and wondering if you have a roadmap of features in mind.

Thanks for any details you share.

**Comment by cubiq** [comment]
I will post a roadmap soon with the planned features. The bottom line is that you should use Comfy :smile:, there's nothing to see here.

If you are still here, I'm trying to build a tool that:
- is based on diffusers
- multi-gpu by default
- multi-hosts by default
- has a lower abstraction needing less nodes
- very easy to build modules for
- very easy to build any kind of custom UI components
- has by default a very powerful and easy to use UI API (auto-complete components, dropdown helpers, bezier curves, ...)
- allows "appifying" a workflow
- has an integrated gallery with advanced search (by model used, by prompt, time, etc...)
- integrated "canvas" tool for better interaction with images
- allows modules isolation
- ...

but mainly it's just because I can more than I should

**Reply by ConorSAI** [reply]
I totally get it,  the module isolation is huge,  so many projects are made with a frustrating array of package versions this does sound appealing.  I am still using comfy of course but will check back from time to time

**Comment by IceMasterT** [comment]
Mellon
